theano/sandbox/cuda/tests/test_cusolver.py

```diff
@@ -0,0 +1,40 @@
+import numpy
+from nose.plugins.skip import SkipTest
+from numpy.testing import assert_allclose
+try:
+    from scipy.linalg.lapack import dgeqrf
+except ImportError:
+    raise SkipTest('SciPy not available')
+
+import theano
+from theano.sandbox.cuda.cusolver import geqrf, cusolver_available
+from theano.sandbox.cuda import cuda_available, CudaNdarray
+
+if not cuda_available:
+    raise SkipTest('CUDA not available')
+if not cusolver_available:
+    raise SkipTest('cuSOLVER not available')
+
+
+def test_geqrf():
+    rng = numpy.random.RandomState(1)
```

Comment:
Could you use `theano.tests.unittest_tools.fetch_seed()` as seed instead of 1? That way it is easier to run the whole test suite with a different seed.